frontend/src/components/DonorsCard/DonorsForm.tsx

```diff
@@ -153,12 +142,12 @@ const DonorsForm = (props: any) => {
       onChange={(e:any) => setDonorName(e.target.value)}
       />
 
-    <h2>Entity Type</h2>
+    <h2>Organizational Structure</h2>
```

Comment:
Sorry for the confusion, this one should actually stay as Entity Type, it was only the recipients that needed to be changed :)